vividus-extension-selenium/src/main/java/org/vividus/steps/ui/GenericSetContextSteps.java

```diff
@@ -66,7 +66,7 @@ public void resetAndChangeContextToElement(Locator locator)
      * in scope of the current context.
      * @param locator locator to find an element
      */
-    @When("I change context to element located `$locator` in scope of current context")
+    @When("I change context to element located by `$locator` in scope of current context")
     public void changeContextToElement(Locator locator)
     {
         changeContext(() -> getBaseValidations().assertElementExists(ELEMENT_TO_SET_CONTEXT, locator).orElse(null),
```

Comment:
steps with new syntax shouldn't use deprecated APIs, so if step to deprecate uses deprecated API:
- deprecate old step using Java `@Deprecated`
- log deprecation warning
- add a new implementation of step (with new syntax and without usage of deprecated API)
- ... other actions (docs, tests) are the same as for deprecation via composite.steps

Reply:
done